src/MLabsPAB.hs

```diff
@@ -1,18 +1,18 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
 {-# LANGUAGE DeriveAnyClass #-}
 
 module MLabsPAB (runPAB) where
 
 import Data.Aeson (FromJSON)
-import Data.Proxy (Proxy)
 import MLabsPAB.Server qualified as Server
 import MLabsPAB.Types (PABConfig (..))
 import Network.Wai.Handler.Warp (run)
 import Plutus.PAB.Effects.Contract.Builtin (HasDefinitions)
 import Prelude
 
-runPAB :: (HasDefinitions t, FromJSON t) => PABConfig -> Proxy t -> IO ()
-runPAB pabConf contractDef = do
+runPAB :: forall t. (HasDefinitions t, FromJSON t) => PABConfig -> IO ()
```

Comment:
We usually give our universal quanitifers kind signatures

Reply:
Addressed it.